lib/containers/utils.pm

```diff
@@ -41,20 +43,62 @@ sub test_seccomp {
     }
 }
 
-sub basic_container_tests {
-    my %args    = @_;
-    my $runtime = $args{runtime};
-    die "You must define the runtime!" unless $runtime;
+sub check_docker_firewall {
+    my $container_name = 'sut_container';
+    my $running        = script_output qq(docker ps -a -q | wc -l);
+    validate_script_output('ip a s docker0', sub { /state DOWN/ }) if $running != 0;
```

Comment:
so `$running`  will be not 0 when at least one container is running , right ? Isn't actually `docker0` interface suppose to be UP in that case ?